### ISSUE TITLE
replace call to functions from the string module in test

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5003]
|<img alt="nueces's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| nueces reported|
|-|-|
|Trac ID|trac#5003|
|Type|enhancement|
|Created|2011-03-27 17:41:53Z|
|Branch|https://github.com/twisted/twisted/tree/str-methods-test-5003|

To make the port to python 3k more easy, I'm replacing all the calls to functions from the string module, for their equivalent from the str type object or calling directly to the built-in method of the string variable.

Attachments:

* [test.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ac8/ac826cb617b41ee99a4f00ebfe0fa922be9f5206/ad0eeb2b1cb8d99eaa860c923c254afb4c9b6041.patch) (2399 bytes) - added by nueces on 2011-03-27 17:42:07Z - patch

<details><summary>Searchable metadata</summary>

```
trac-id__5003 5003
type__enhancement enhancement
reporter__nueces nueces
priority__normal normal
milestone__ 
branch__branches_str_methods_test_5003 branches/str-methods-test-5003
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__core core
keywords__py3k py3k
time__1301247713000000 1301247713000000
changetime__1302055770000000 1302055770000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_1"></a>|@lvh commented|
|-|-|

(In [31472]) Branching to 'str-methods-test-5003'

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh set owner to @lvh<br>@lvh set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh commented|
|-|-|

Patch looks good, here's the buildbot:

http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/str-methods-test-5003

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh commented|
|-|-|

Looks good! Thanks for your patch, added to my patch queue for tomorrow.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_5"></a>|@lvh set status to `closed`|
|-|-|

(In [31554]) Replaced calls to functions from the string module in twisted.test.

Author: nueces
Reviewers: lvh
Fixes: [#5003](https://github.com/twisted/twisted/issues/5003)

twisted.test was previously using deprecated calls to functions in the string
module, which are deprecated in favor of str methods.
